05-classification.Rmd

```diff
@@ -963,12 +988,14 @@ for the number of neighbours and weight function (for now, these are just the va
  manually when we created `knn_spec` above). But now the fit object also includes information about
 the overall workflow, including the centering and scaling preprocessing steps.
 
-Let's visualize the predictions that this trained K-nearest neighbour model will make on new observations.
```

Comment:
If we are putting it all together, we should also `predict`, right? This might be the place to make the point that the workflow object bakes the new data!

Reply:
i plan to do this and the below comment (they're interdependent)

Reply:
done